.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,140 @@
+        worker_volume_type: "io1"
+        worker_volume_size: 30
+        worker_iops: 1500
+    - name: Wait Kubernetes control plane is up and running
+      uses: nick-invision/retry@v2
+      with:
+        timeout_seconds: 30
+        max_attempts: 20
+        retry_wait_seconds: 30
+        command: kubectl get --raw='/readyz?verbose'
+      env:
+        KUBECONFIG: rke-cluster-for-cluster-api/rke/kube_config_cluster.yml
+    - name: Wait all infrastructure app are really up and running
+      uses: jupyterhub/action-k8s-await-workloads@v1
+      with:
+        workloads: ""
+        namespace: "" 
+        timeout: 600
+        max-restarts: -1
+      env:
+        KUBECONFIG: rke-cluster-for-cluster-api/rke/kube_config_cluster.yml
+    - name: Deploy Docker-registry into cluster
+      run: |
+        ansible-playbook addons/docker-registry/playbook.yaml
+        ./addons/docker-registry/start_port_forwarding.sh &
+      env:
+        KUBECONFIG: rke/kube_config_cluster.yml
+        ANSIBLE_CONFIG: ansible.cfg
+      working-directory: rke-cluster-for-cluster-api
+    - name: Wait to access registry
+      uses: nick-invision/retry@v2
+      with:
+        timeout_seconds: 30
+        max_attempts: 10
+        retry_wait_seconds: 1
+        command: curl http://127.0.0.1:4242/v2
+    - name: Build and Push Docker image 
+      run: |
+        make docker-build
+        make docker-push
+        docker image prune -a -f
+      env:
+        IMG: 127.0.0.1:4242/cluster-api-outscale-controller:${{ github.sha }}
+        DOCKER_BUILDKIT: 1
+    - name: Retrieve NodePort Ip and NodePort
+      run: |
+        echo 'NODEPORT_PORT<<EOF' >> $GITHUB_ENV
+        echo "$(kubectl get --namespace default -o jsonpath='{.spec.ports[0].nodePort}' services docker-registry)" >> $GITHUB_ENV
+        echo 'EOF' >> $GITHUB_ENV
+        echo 'NODEPORT_IP<<EOF' >> $GITHUB_ENV
+        echo "$(kubectl get nodes --namespace default -o jsonpath='{.items[0].status.addresses[0].address}')"  >> $GITHUB_ENV
+        echo 'EOF' >> $GITHUB_ENV
+      env:
+        KUBECONFIG: rke-cluster-for-cluster-api/rke/kube_config_cluster.yml
+    - name: Create ns and set credential
+      run: make credential
+      shell: bash
+      env:
+         KUBECONFIG: rke-cluster-for-cluster-api/rke/kube_config_cluster.yml
+         OSC_ACCESS_KEY: ${{secrets.OSC_ACCESS_KEY}}
+         OSC_SECRET_KEY: ${{secrets.OSC_SECRET_KEY}}
+    - name: Launch conformance
+      run:  KUBECONFIG=$GITHUB_WORKSPACE/rke-cluster-for-cluster-api/rke/kube_config_cluster.yml make e2econformance
+      shell: bash
+      id: e2etest
+      env:
+        IMG: ${{ env.NODEPORT_IP }}:${{ env.NODEPORT_PORT }}/cluster-api-outscale-controller:${{ github.sha }}
+        OSC_ACCESS_KEY: ${{secrets.OSC_ACCESS_KEY}}
+        OSC_SECRET_KEY: ${{secrets.OSC_SECRET_KEY}}
+    - name: Get cluster api outscale logs
+      run: make logs-capo
+      if: ${{ steps.e2etest.outcome == 'success' }}
```

Comment:
You only want the logs when it is a success ?

Reply:
Ok change